types/get-pixels/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for get-pixels 3.3
+// Project: https://github.com/scijs/get-pixels#readme
+// Definitions by: Don McCurdy <https://github.com/donmccurdy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import ndarray = require('ndarray');
+
+type Callback = (err: string | null, pixels: ndarray | null) => void;
```

Comment:
```ts
type Callback = (err: string | null, pixels: ndarray | null) => void;
```
the callback seems to support `Error`:
https://github.com/scijs/get-pixels/blob/master/node-pixels.js#L137

Reply:
Thanks! Can't see anywhere using a string after all, so I've made this `Error | null`.